spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowControllerAutoConfiguration.java

```diff
@@ -363,7 +365,8 @@ public StreamDeploymentController updatableStreamDeploymentController(
 
 		@Bean
 		public SkipperClient skipperClient(SkipperClientProperties properties,
-				RestTemplateBuilder restTemplateBuilder, ObjectMapper objectMapper) {
+				RestTemplateBuilder restTemplateBuilder, ObjectMapper objectMapper,
+				@Autowired(required = false) OAuth2TokenUtilsService oauth2TokenUtilsService) {
```

Comment:
Same `Optional` comment here vs using `@Autowired`.

Reply:
Changed to `@Nullable`